packages/now-node/src/typescript.ts

```diff
@@ -191,10 +191,11 @@ export function register(opts: Options = {}): Register {
       return;
     }
     const error = createTSError(diagnostics);
-    // Print error in red color and continue execution.
-    console.error('\x1b[31m%s\x1b[0m', error);
     if (shouldExit) {
-      process.exit(1);
+      throw error;
+    } else {
+      // Print error in red color and continue execution.
+      console.error('\x1b[31m%s\x1b[0m', error);
     }
```

Comment:
Is it possible to add a test for this?

Reply:
I would prefer to delegate writing tests to code owners. I am not familiar with testing environment. Please consider this RS as an issue.

Reply:
Do you have a failing deployment? How did you know this needed to be changed?

Reply:
Yeah, https://zeit.co/zeit/igorklopov-tests/bom71ibn3/source?f=src%2Fsrc%2Findex.ts

Reply:
What is abnormal about this deployment? I see the error message: 

```
Error: src/index.ts(1,24): error TS2345: Argument of type '"42"' is not assignable to parameter of type 'number'.
```